### PR TITLE
translating pilot into Chinese

### PR DESCRIPTION
**Pull request by rootsongjc**
- this page has been translated before, I just review it again

@geeknoid If the CI build successfully, you can merge it directly.

### CODE REVIEW

File: content_zh/docs/concepts/traffic-management/pilot.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+Pilot 负责部署在 Istio 服务网格中的 Envoy 实例的生命周期管理。
+
+{{< image width="60%" ratio="72.17%"
+    link="../img/pilot/PilotAdapters.svg"
+    caption="Pilot 架构"
+    >}}
+
+如上图所示，Pilot 维护了网格中的服务的规范表示，这个表示是独立于底层平台的。Pilot 中的平台特定适配器负责适当填充此规范模型。例如，Pilot 中的 Kubernetes 适配器实现必要的控制器来 watch Kubernetes API server 中 pod 注册信息、ingress 资源以及用于存储流量管理规则的第三方资源的更改。该数据被翻译成规范表示。Envoy 特定配置是基于规范表示生成的。
+
+Pilot 公开了用于[服务发现](https://envoyproxy.github.io/envoy/configuration/cluster_manager/sds_api.html) 、[负载均衡池](https://envoyproxy.github.io/envoy/configuration/cluster_manager/cds.html)和[路由表](https://envoyproxy.github.io/envoy/configuration/http_conn_man/rds.html)的动态更新的 API。这些 API 将 Envoy 从平台特有的细微差别中解脱出来，简化了设计并提升了跨平台的可移植性。
```
Comment (fleeto) [inline content_zh/docs/concepts/traffic-management/pilot.md]: Pilot 公开了用于[服务发现](https://www.envoyproxy.io/docs/envoy/latest/api-v1/cluster_manager/sds) 、[负载均衡池](https://www.envoyproxy.io/docs/envoy/latest/api-v1/cluster_manager/cds)和[路由表](https://www.envoyproxy.io/docs/envoy/latest/api-v1/route_config/rds)的动态更新的 API。这些 API 将 Envoy 从平台特有的细微差别中解脱出来，简化了设计并提升了跨平台的可移植性。

Comment (fleeto) [inline content_zh/docs/concepts/traffic-management/pilot.md]: Pilot 公开了用于[服务发现](https://www.envoyproxy.io/docs/envoy/latest/api-v1/cluster_manager/sds) 、[负载均衡池](https://www.envoyproxy.io/docs/envoy/latest/api-v1/cluster_manager/cds)和[路由表](https://www.envoyproxy.io/docs/envoy/latest/api-v1/route_config/rds)的动态更新的 API。这些 API 将 Envoy 从平台特有的细微差别中解脱出来，简化了设计并提升了跨平台的可移植性。

Comment (gyliu513) [reply content_zh/docs/concepts/traffic-management/pilot.md]: +1 and please keep consistent with https://github.com/istio/istio.github.io/blob/master/content/docs/concepts/traffic-management/pilot.md

Comment (rootsongjc) [reply content_zh/docs/concepts/traffic-management/pilot.md]: I have updated the links as the original

**Comment by rootsongjc** [comment]
All error fixed, it's OK to be merge.

**Pull request merged**